tests/Server/Middleware/RequireRoleTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+use Carbon\Carbon;
+use DoSomething\Gateway\Server\Token;
+use DoSomething\Gateway\Server\Middleware\RequireRole;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+class RequireRoleTest extends TestCase
+{
+    /** @test */
```

Comment:
I don't think you need the `/** @test */` comment if the method begins with `test...()`. That being said didn't we move to not naming tests w/ prefix `test` and want to move to `it_can_` or whatever? Also not sure if makes sense to change if rest of package already does `test` prefix. Just random thoughts 🤔

Reply:
Yeah I think you're right! I was considering writing these as `it_should_...` style, but decided it made a bit more sense to hold off & maybe do them all at once in another PR.